addons/binding/org.openhab.binding.feican/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="feican"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <channel-type id="switch">
+        <item-type>Switch</item-type>
+        <label>Switch</label>
+        <description>TThis channel supports switching the light on and off.
+        </description>
+        <category>Light</category>
+        <tags>
+            <tag>Lighting</tag>
+        </tags>
+    </channel-type>
+
+    <channel-type id="color">
+        <item-type>Color</item-type>
+        <label>Color</label>
+        <description>This channel supports adjusting the color of a light.
+        </description>
+        <category>ColorLight</category>
+        <tags>
+            <tag>Lighting</tag>
+        </tags>
+    </channel-type>
+
+    <channel-type id="color_temperature">
+        <item-type>Dimmer</item-type>
+        <label>Color Temperature</label>
+        <description>This channel supports adjusting the color temperature from cold (0%) to warm (100%).</description>
+        <category>ColorLight</category>
+    </channel-type>
+
+    <channel-type id="brightness">
+        <item-type>Dimmer</item-type>
+        <label>Brightness</label>
+        <description>This channel supports adjusting the brightness of the color, color temperature of program.</description>
+        <category>ColorLight</category>
+    </channel-type>
+
+    <channel-type id="white_brightness">
```

Comment:
Please remove this - as far as I understand, there is no functional difference to "brightness" from a user perspective.

Reply:
The 'normal' brightness sets the brightness on color. When using white_brightness the device acts as a white bulb. So when setting the white_brightness it will switch to a white color with the given brightness. So it's functionally different.

Reply:
You should keep the modelling identical to hue/lifx/tradfri and all the others.
Switching from color to white when brightness is adjusted is imho no valid functionality.
If you have a brightness channel, it should only change the brightness, no matter what color is currently active.

Reply:
I'd like to see it as this device are 2 devices in 1: a Dimmable Light and a Color Light. For hue there is a brightness channel only for dimmable hue lights and the channel is not available for color lights. 
I thought it was a nice feature to also be able to just use it as a dimmable light and because it was supported by the device itself this way. But if you think it's confusing I'll remove it.

Reply:
Well, after all, it is just a single device. And the user can easily use it as a white bulb by linking the color channel to a Dimmer item - and by choosing the warmth of the white through the color temperature channel. So yes, since we have all similar bulbs done in that way, I would very much prefer to remove this channel here, so that it is the same way for the users in all bindings. Thanks for your understanding 🙇